Kitodo/src/test/java/org/kitodo/production/forms/ProcessFormIT.java

```diff
@@ -104,9 +104,8 @@ public void testProcessesSelection() throws Exception {
         Assert.assertEquals(new ArrayList<>(Arrays.asList(1, 2, 5)), selectedIds);
 
         processForm.setAllSelected(false);
-        processForm.selectedProcessesOrProcessDTOs = ServiceManager.getProcessService().findByAnything("SelectionTest");
         selectedIds = processForm.getSelectedProcesses()
                 .stream().map(Process::getId).sorted().collect(Collectors.toList());
-        Assert.assertEquals(new ArrayList<>(Arrays.asList(4, 5)), selectedIds);
+        Assert.assertEquals(new ArrayList<>(Arrays.asList(1, 2, 5)), selectedIds);
```

Comment:
It is because, above, the line
``processForm.selectedProcessesOrProcessDTOs = ServiceManager.getProcessService().findByAnything("SelectionTest");``
goes out. There is no function `findByAnything(String)` no more. However, I wanted to keep this test working, because selecting / unselecting should work even without index, so I removed this and changed the test accordingly. There can be other tests for testing the index, but a selecting test should test selecting. It is clearer this way.